src/query/ast/src/ast/statements/replace.rs

```diff
@@ -54,18 +54,18 @@ impl Display for ReplaceStmt {
         if !self.columns.is_empty() {
             write!(f, "(")?;
             write_comma_separated_list(f, &self.columns)?;
-            write!(f, ") ")?;
+            write!(f, ")")?;
         }
-        write!(f, "ON CONFLICT")?;
+        write!(f, " ON CONFLICT")?;
         if !self.on_conflict_columns.is_empty() {
             write!(f, "(")?;
             write_comma_separated_list(f, &self.on_conflict_columns)?;
             write!(f, ") ")?;
```

Comment:
write!(f, ") ")?;

modify to 

```
write!(f, " )")?;
```

Reply:
Let me fix this in next pr.

Because the current behavior is wrong, this is a fix.